setup.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
-apt-get update >/dev/null
-apt-get -y install linux-headers-$(uname -r)
-mount -t debugfs debugfs /sys/kernel/debug
+DEBUG_PATH=/sys/kernel/debug
+
+apt-get update
```

Comment:
Any particular reason behind the removal of `> /dev/null`?

I originally put it in because it will just spam syslog with a bunch of stuff that's not relevant but I'm not married to the decision either way.

Reply:
Hm, no particular reason. I think I had originally removed it to see what would happen with the update (just to get feedback it was working) but I forgot to put the `/dev/null` back.